scripts/run-ginkgo-integration-suite.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# This script can be used to run any ginkgo integration suite using SUITE_NAME environment variable.
+# Prerequisite: cluster with at least three Linux nodes must already exist.
+# It is up to the caller to ensure that tests are valid to run against cluster, i.e. IPv6 tests require IPv6 cluster.
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+GINKGO_TEST_BUILD="$SCRIPT_DIR/../test/build"
+SKIP_MAKE_TEST_BINARIES=1
+
+source "$SCRIPT_DIR"/lib/add-on.sh
+source "$SCRIPT_DIR"/lib/cluster.sh
+source "$SCRIPT_DIR"/lib/canary.sh
+
+function run_ginkgo_test() {
+  (CGO_ENABLED=0 ginkgo $EXTRA_GINKGO_FLAGS -v --timeout 30m --no-color --fail-on-pending $GINKGO_TEST_BUILD/$SUITE_NAME.test -- --cluster-kubeconfig="$KUBE_CONFIG_PATH" --cluster-name="$CLUSTER_NAME" --aws-region="$REGION" --aws-vpc-id="$VPC_ID" --ng-name-label-key="kubernetes.io/os" --ng-name-label-val="linux")
+}
+
+load_cluster_details
+load_addon_details
+
+echo "Running $SUITE_NAME integration tests against the latest VPC CNI addon version"
+
+install_add_on "$LATEST_ADDON_VERSION"
```

Comment:
*important* - We can leave out add-on installation from this script. Let the script's focus only be running the tests. Doing add-on installation was conflating the responsibilities (even earlier) So we can remove this.

Reply:
Yep, leaving it to the one creating/configuring the cluster makes sense to me. Removed in latest revision.